src/testmod/java/dev/emi/trinkets/TrinketsTest.java

```diff
@@ -13,14 +17,26 @@ public class TrinketsTest implements ModInitializer {
 	public static final String MOD_ID = "trinkets-testmod";
 	public static final Logger LOGGER = LogManager.getLogger();
 	public static Item TEST_TRINKET;
+	public static Item TEST_TRINKET_2;
 
 	@Override
 	public void onInitialize() {
 		LOGGER.info("[Trinkets Testmod] test mod was initialized!");
 		TEST_TRINKET = Registry.register(Registries.ITEM, identifier("test"), new TestTrinket(new Item.Settings().maxCount(1).maxDamage(100)));
+		TEST_TRINKET_2 = Registry.register(Registries.ITEM, identifier("test2"), new TestTrinket2(new Item.Settings().maxCount(1)));
+		TrinketEquipCallback.EVENT.register(((stack, slot, entity) -> {
+			if(stack.isOf(TEST_TRINKET_2)){
+				entity.getWorld().playSound(null,entity.getBlockPos(), SoundEvents.ENTITY_ARROW_HIT, SoundCategory.PLAYERS, 1f,1f);
```

Comment:
Please have spaces between args here and below.

Reply:
Ok